ppr-ui/src/components/collateral/generalCollateral/GeneralCollateral.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-container v-if="summaryView" style="padding: 16px 12px 0 30px;">
+    <gen-col-summary />
+  </v-container>
+  <v-container v-else class="pa-0">
+    <gen-col-edit
+      :showInvalid="showInvalid"
+      @valid="emitValid"
+    />
+  </v-container>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  reactive,
+  toRefs
+} from '@vue/composition-api'
+// local components
+import { GenColEdit, GenColSummary } from '.'
+// local types/helpers/etc.
+import { APIRegistrationTypes } from '@/enums' // eslint-disable-line no-unused-vars
+
+export default defineComponent({
+  name: 'GeneralCollateral',
+  components: {
+    GenColEdit,
+    GenColSummary
+  },
+  props: {
+    isSummary: {
+      type: Boolean,
+      default: false
+    },
+    setRegistrationType: String as () => APIRegistrationTypes,
```

Comment:
Is this used?

Reply:
yep

Reply:
it needs to get passed to the child. I think ideally none of the children would need to access the store at all, but that would've been too complicated to alter this time